data/wv/retired/Mark-Hunt-0f9b6e27-110c-49ed-acb5-6bc10cab45d7.yml

```diff
@@ -1,27 +0,0 @@
-id: ocd-person/0f9b6e27-110c-49ed-acb5-6bc10cab45d7
-name: Mark Hunt
-given_name: Mark
-family_name: Hunt
-image: http://www.legis.state.wv.us/../images/members/2013/house/hunt_mark.jpg
```

Comment:
@sroomf undelete and move to leg. We also need to add an end date of 12/01/2022 to his current party and add a new party for Republican. We also need to add an additional role:
- start_date: '2022-12-01'
  type: upper
  jurisdiction: ocd-jurisdiction/country:us/state:wv/government
  district: '8'